sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Specialized;
 using System.Threading;
 using System.Web;
+
```

Comment:
hmm does the "azp" command not work for container registry, "azp run net - containerregistry - ci"

Reply:
you need a slash '/azp run net - core - ci'

Reply:
Updated